uvicorn/protocols/http/httptools_impl.py

```diff
@@ -389,11 +399,11 @@ def __init__(
         # Response state
         self.response_started = False
         self.response_complete = False
-        self.chunked_encoding = None
+        self.chunked_encoding: Optional[bool] = None
         self.expected_content_length = 0
 
     # ASGI exception wrapper
-    async def run_asgi(self, app):
+    async def run_asgi(self, app: Callable) -> None:
```

Comment:
Too generic. One of my PRs have this already, or even the current one already has? Not sure.